### PR TITLE
change header bar for adapt it with the current environnement

### PR DESCRIPTION
**Pull request by Debaerdm**
# Pull Request Template

## PR Description

This feature was maked for adapt the header bar with different environnemet (here macos and windows), for example:

Windows (Dark): 

image

Windows (Light):

image

macos (Dark): 

image

macos (Light):

image

### CODE REVIEW

File: public/global.css [hunk 1]
```diff
@@ -58,6 +58,10 @@ input:disabled {
 	background: #ccc;
 }
 
+::-webkit-scrollbar-corner {
+	background-color: #fff;
```
Comment (itupix) [inline public/global.css:62]: Dark mode et light mode ?

File: src/components/Header/MacosBar.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		display: none;
+	}
+}
+
+.window-controls:hover svg {
+	display: flex;
+}
+
+.close-button {
+	background-color: #fe6057;
```
Comment (itupix) [inline src/components/Header/MacosBar.scss:29]: You should create variables for all those colors

File: src/components/Header/WindowsBar.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	transition: 200ms;
+
+	& path {
+		transition: 200ms;
+		transition-timing-function: ease-in-out;
+	}
+}
+
+.close-button:hover {
+	background-color: red;
```
Comment (itupix) [inline src/components/Header/WindowsBar.scss:22]: True Windows red ?

File: public/global.css (hunk 1 above)
Comment (Debaerdm) [reply public/global.css:62]: For dark mode it's in the end of file: 

::-webkit-scrollbar-corner {
	background-color: #444;
}

File: src/components/Header/WindowsBar.scss (hunk 3 above)
Comment (Debaerdm) [reply src/components/Header/WindowsBar.scss:22]: I think i'm going to use color picker for that

**Pull request merged**